src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -5000,5 +5000,26 @@ partial class C
                 //     public partial int Prop2 { get; set { } }
                 Diagnostic(ErrorCode.ERR_ConcreteMissingBody, "get").WithArguments("C.Prop2.get").WithLocation(7, 32));
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/74679")]
+        public void WRN_SequentialOnPartialClass_NotReportedForPartialProperty()
```

Comment:
Please add a version of this test that looks like this:

```cs
                partial struct S
                {
                    partial int I { get; }
                }

                partial struct S
                {
                    public S() => i = 42;
                }

                partial struct S
                {
                    private readonly int i;
                    partial int I => i;
                }
```
Probably also worth adding a note to the field test plan, cc @cston.

Reply:
#57012 says:
> Check warning WRN_SequentialOnPartialClass is produced when needed